app/models/setting/content.rb

```diff
@@ -157,7 +157,9 @@ def self.default_settings
       self.set('autosearch_delay', N_('If Autosearch is enabled, delay in milliseconds before executing searches while typing.'),
                500, N_('Autosearch delay')),
       self.set('bulk_load_size', N_('The number of items fetched from a single paged Pulp API call.'), 2000,
-               N_('Pulp bulk load size'))
+               N_('Pulp bulk load size')),
+      self.set('upload_profiles_without_dynflow', N_('Allow Candlepin profile uploads to use Rails directly instead of being wrapped in Dynflow actions'), true,
```

Comment:
The description of this is too low level as far as the user is concerned. We might word it to articulate when they might change the setting

Reply:
@jturel I changed it but it's still quite wordy. open to suggestions (also @parthaa)